openmetadata-ui/src/main/resources/ui/src/jsons/en.ts

```diff
@@ -27,11 +27,12 @@ const jsonData = {
 
     'delete-glossary-error': 'Error while deleting glossary!',
     'delete-glossary-term-error': 'Error while deleting glossary term!',
-    'delete-team-error': 'Error while deleting team!',
+    'delete-ingestion-error': 'Error while deleting ingestion workflow',
```

Comment:
```suggestion
    'delete-ingestion-error': 'Error while deleting ingestion workflow!',
```

Reply:
This error message has some dynamic suffixes